2-mandatory/3-stocks.js

```diff
@@ -34,9 +34,31 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+
+    let averagePrices = [];
+    for (prices of closingPricesForAllStocks) {
+        sum = 0;
+        for (item of prices) { //we use for within for so we can access the arrays of the array
+            sum += parseFloat(item);
```

Comment:
Might be worth double-checking - is `item` already a Number? If it is, we may not need `parseFloat` here.